### PR TITLE
Add World Health Organisation & ZOA to the GB Dashboard - remove AFD

### PR DESCRIPTION
**Pull request by dalepotter**
Adds these two organisations to the GB dashboard.

### CODE REVIEW

File: data/static/base_info.csv [hunk 1]
```diff
@@ -1,4 +1,5 @@
 name_en,first_published,org_type,registry_id,baseline
+Agence Française de Développement,30-09-2016,Other Public Sector,afd,0
```
Comment (hayfield) [inline data/static/base_info.csv]: The date should be in the format `yyyy-mm-dd`

File: data/static/base_info.csv [hunk 2]
```diff
@@ -61,7 +62,9 @@ United Nations World Food Programme (WFP),2013-06-11,Multilateral,wfp,70
 United States,2013-01-02,Government,unitedstates,27.25
 US Agency for International Development (USAID),2017-04-27,Government,usaid,33
 "World Bank, The",2011-03-14,Multilateral,worldbank,51.35
+World Health Organisation,2017-06-08,Multilateral,who,0
 World Vision International (WVI),2015-10-16,International NGO,wvi,6.25
 World Vision Netherlands,2016-06-20,National NGO,wvnld,3.25
 World Vision UK,2012-06-13,International NGO,wvuk,22
 World Vision Zimbabwe,2017-12-05,International NGO,wvz-gec,0
+ZOA,10-02-2016,International NGO,zoa,0
```
Comment (hayfield) [inline data/static/base_info.csv]: The date should be in the format `yyyy-mm-dd`

**Review by hayfield — DISMISSED** [review]
A couple of mis-formatted dates

File: data/static/base_info.csv (hunk 1 above)
Comment (PetyaKangalova) [reply data/static/base_info.csv]: Thanks! Just to say we are also holding off adding AFD for now.

File: data/static/base_info.csv (hunk 2 above)
Comment (PetyaKangalova) [reply data/static/base_info.csv]: Have added Zoa to title of the pull request to avoid confusion.

**Comment by dalepotter** [comment]
This PR was closed automatically as an unintended consequence of the set-up of the `dev` branch without phase 1 changes. It has now been reopened.

File: data/static/base_info.csv (hunk 2 above)
Comment (dalepotter) [reply data/static/base_info.csv]: Date formatting fixed in 9f5a055.

File: data/static/base_info.csv (hunk 1 above)
Comment (dalepotter) [reply data/static/base_info.csv]: Date formatting fixed in 9f5a055 (although I have received notification from @PetyaKangalova that AFD will be removed from the Grand Bargain signatories).

File: data/static/base_info.csv [hunk 3]
```diff
@@ -61,7 +62,9 @@ United Nations World Food Programme (WFP),2013-06-11,Multilateral,wfp,70
 United States,2013-01-02,Government,unitedstates,27.25
 US Agency for International Development (USAID),2017-04-27,Government,usaid,33
 "World Bank, The",2011-03-14,Multilateral,worldbank,51.35
+World Health Organisation,2017-06-08,Multilateral,who,0
```
Comment (PetyaKangalova) [inline data/static/base_info.csv:69]: Waiting for  confirmation on when WHO formally joined the Grand Bargain as a signatory

File: data/static/base_info.csv [hunk 4]
```diff
@@ -1,4 +1,5 @@
 name_en,first_published,org_type,registry_id,baseline
+Agence Française de Développement,2016-09-30,Other Public Sector,afd,0
```
Comment (PetyaKangalova) [inline data/static/base_info.csv]: France is a signatory but AFD are not publishing humanitarian data so will not be added to the dashboard. Confirmation received from AFD.

File: data/static/base_info.csv [hunk 5]
```diff
@@ -61,7 +62,9 @@ United Nations World Food Programme (WFP),2013-06-11,Multilateral,wfp,70
 United States,2013-01-02,Government,unitedstates,27.25
 US Agency for International Development (USAID),2017-04-27,Government,usaid,33
 "World Bank, The",2011-03-14,Multilateral,worldbank,51.35
+World Health Organisation,2017-06-08,Multilateral,who,0
 World Vision International (WVI),2015-10-16,International NGO,wvi,6.25
 World Vision Netherlands,2016-06-20,National NGO,wvnld,3.25
 World Vision UK,2012-06-13,International NGO,wvuk,22
 World Vision Zimbabwe,2017-12-05,International NGO,wvz-gec,0
+ZOA,2016-02-10,International NGO,zoa,0
```
Comment (PetyaKangalova) [inline data/static/base_info.csv]: Joined at the end of July 2017. Baseline value should be recalculated for 1 August 2017. It has been agreed that the first of the month will be taken for any new signatories.

File: data/static/base_info.csv (hunk 3 above)
Comment (PetyaKangalova) [reply data/static/base_info.csv:69]: Confirmation from WHO that they joined  1 Jan 2018 . Please recalculate baseline as of 1 January 2018.

**Review by PetyaKangalova — APPROVED** [review]
All baseline values are correct as per @wendyrogers 's instructions

**Pull request merged**